bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormStructureParserImpl.java

```diff
@@ -85,6 +91,9 @@ private String getPropertyFromFormContainer(@Nullable Resource resource, @NotNul
     }
 
     private String getFormContainerPath(Resource resource) {
+        if (request != null && request.getAttribute("formContainerPath") != null) {
```

Comment:
Why haven't we use constant for this ? i think I commented in my last review. REQ_ATTR prefix